src/listAllFixtures.ts

```diff
@@ -31,7 +31,7 @@ const isFixture = (absolutePath: string): boolean => {
 export const listDirectoryFixtures = (basePath: string): FixturesMap =>
  walkSync(basePath)
   .filter(isFixture)
-  .reduce((acc, absolute) => ({...acc, [path.relative(basePath, absolute)]: absolute}), {});
+  .reduce((acc, absolute) => ({...acc, [path.relative(basePath, absolute).replace(/\\/g, '/')]: absolute}), {});
```

Comment:
How it works? Probably same as https://github.com/blueberryapps/vcr.js/pull/10. In #6 windows will try to use path with slashes instead of backslashes, isn't it a problem? @ondrejbartas